packages/amplify_core/lib/amplify_core.dart

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: 
```suggestion
 * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Reply:
Made this 2020 since this file was added in 2020. Not sure if that actually matters.